### DISCUSSION TITLE
Typo in French core resource properties in code base

### DISCUSSION THREAD

**Discussion by alexgille** [dbeaver/dbeaver#19034] [DBeaver Development]
Hi all,

I found an incorrect wording/sentence in the code base for the French language and I wish to contribute by fixing it. I understood that such changes were not really accepted on this repository but still I want to confirm this first.

How can I contribute in this way?

Thanks
Regards

**Comment by LonwoLonwo** [comment]
Hello @alexgille 

Like usually. You must create DBeaver fork, add your changes, commit and push them, and then create PR to the main code.

**Comment by E1izabeth** [comment]
This wiki might be helpful https://github.com/dbeaver/dbeaver/wiki/Localization

**Comment by mkgrgis** [comment]
@E1izabeth, do You know, will properties like [this](https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.data.gis.view/OSGI-INF/l10n/bundle_ru.properties) converted to simple UTF-8?

**Reply by E1izabeth** [reply]
It's native2ascii conversion
